mllib/src/main/scala/org/apache/spark/mllib/clustering/LDAModel.scala

```diff
@@ -345,6 +346,12 @@ class DistributedLDAModel private (
     }
   }
 
+  /** Java-friendly version of [[topicDistributions]] */
+  def javaTopicDistributions: JavaPairRDD[java.lang.Long, Vector] = {
+    new JavaPairRDD[java.lang.Long, Vector](
```

Comment:
They are the same. Java compiler can infer the type in `fromRDD` so you don't have to write `[java.lang.Long, Vector]` three times.